app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -381,10 +384,22 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
                             .setOnClickListener(new View.OnClickListener() {
                                 @Override
                                 public void onClick(View view) {
-                                    Intent i = new Intent(Intent.ACTION_VIEW);
-                                    i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                                    i.setData(Uri.parse(url));
-                                    zulipApp.startActivity(i);
+                                    if (Build.VERSION.SDK_INT < 15)
+                                    {
+                                        Intent i = new Intent(Intent.ACTION_VIEW);
+                                        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                                        i.setData(Uri.parse(url));
+                                        zulipApp.startActivity(i);
+                                    }
+                                    else
+                                    {
+                                        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+                                        builder.setStartAnimations(zulipApp.getZulipActivity(), R.anim.slide_in_right, R.anim.slide_out_left);
```

Comment:
@vishwesh3 android inbuit transition don't have transition like `android.R.anim.slide_in_right` and `android.R.anim.slide_out_left` . So eventually i need this anim xml file to do so. 
If i'm missing out something then do correct me. :)